test/identity-provider.test.js

```diff
@@ -226,11 +214,11 @@ describe('Identity Provider', function() {
 
       let err
       try {
-        const signature = await identity.provider.sign(modifiedIdentity, data, keystore)
+        const signature = await identity.provider.sign(modifiedIdentity, data, keystore) /* eslint-disable-line */
```

Comment:
Instead of `disable-line` lint comment, let's fix this as per (what I assume the linter is erroring for) the linter error and change the line to:
```js
await identity.provider.sign(modifiedIdentity, data, keystore)
```

(remove the assignment to variable)